### ISSUE TITLE
[rate-limit] missing track rule

### ISSUE THREAD

**Issue by bedis** [haproxytech/kubernetes-ingress#24]
The configuration generated by the rate-limit feature misses a "track" rule, which means it currently does not rate-limit anything:

```
  acl ratelimit_cnt_abuse src_get_gpc0(RateLimit) gt 0
  acl ratelimit_inc_cnt_abuse src_inc_gpc0(RateLimit) gt 0
  acl ratelimit_is_abuse src_http_req_rate(RateLimit) ge 10
  http-request deny deny_status 0 if ratelimit_is_abuse ratelimit_inc_cnt_abuse
  http-request deny deny_status 0 if ratelimit_cnt_abuse
```

**Comment by bedis** [comment]
So, it seems the controller properly set the Configuration structure:
```TCPRequests:map[rate-limit:[{Action:track-sc0 src table RateLimit Cond: CondTest: ID:0xc0003c7670```

The problem may lie below, in the client-native library

**Comment by bedis** [comment]
Here is the relevant PR: https://github.com/haproxytech/kubernetes-ingress/pull/26

**Comment by bedis** [comment]
Fixed with https://github.com/haproxytech/kubernetes-ingress/commit/dd2386d74d7b237bcde266b8e03d86b9fc6a98be.
Available in version 1.1.2 and above.
